webapp/src/components/properties/user/user.tsx

```diff
@@ -16,24 +16,23 @@ type Props = {
 }
 
 const UserProperty = (props: Props): JSX.Element => {
-    const workspaceUsers = useContext(WorkspaceUsersContext) as Array<IUser>|undefined
-    const workspaceUsersById = useContext(WorkspaceUsersByIDContext) as Map<string, IUser>|undefined
+    const workspaceUsers = useContext(WorkspaceUsersContext) as WorkspaceUsersContextData
```

Comment:
can it be `useContext<WorkspaceUsersContextData>(WorkspaceUsersContext)` instead?

Reply:
You learn something new everyday! Thanks for the tip! Done.